erdjava/src/main/java/elrond/Bech32.java

```diff
@@ -0,0 +1,299 @@
+/*
```

Comment:
Is there a reason why we didn't just used https://github.com/bitcoinj/bitcoinj/blob/master/core/src/main/java/org/bitcoinj/core/Bech32.java?

Reply:
Used it in the beginning. The linter was somehow complaining, and I had to do changes anyway - I would had to then describe the changes in the header anyway (with respect to the Apache License). Furthermore, there's no reference Java implementation, neither of the implementations (PR) were accepted by Pieter Wuille:

https://github.com/sipa/bech32/pulls

So I took the reference implementation in Python and started to translate it.

But yes, we now have `bitcoinj` as a dependency for generating mnemonics, so... Changed to use Bech32 as well.

Moved my code to the special repository called "nice code that didn't make it" :)